packages/memory/src/index.ts

```diff
@@ -630,16 +630,21 @@ export class Memory extends MastraMemory {
 
   public async getUserContextMessage({ threadId }: { threadId: string }) {
     const workingMemory = await this.getWorkingMemory({ threadId });
+
     if (!workingMemory) {
       return null;
     }
 
-    return `The following is the most up-to-date information about the user's state and context:
+    return `You have the following information about the user's state and context. It is vital that you use this as the source of truth when generating responses.
+    
+<working_memory>
 ${JSON.stringify(workingMemory)}
-Use this information as the source of truth when generating responses. 
-Do not reference or mention this memory directly to the user. 
-If conversation history shows information that is not in the working memory, use the working memory as the source of truth.
-If there is a discrepancy between this information and conversation history, always rely on this information unless the user explicitly asks for an update.
+</working_memory>
+
+Guidelines:
+- Do not reference or mention this information directly to the user.
```

Comment:
I think we should tweak this since the agent should mention the info, they just shouldn't refer to "working memory"
```suggestion
- Do not mention your memory system directly to the user, they know nothing about it, refer to this information as facts you have remembered.
```